test/browser_tests/page_components/footer.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function Footer() {
+  this.get = function() {
+    browser.get( '/' );
```

Comment:
That is correct. I was thinking that we would have component objects to represent things we are using across multiple pages. Things like primary nav, secondary nav, etc... which aren't tied to a particular page.